src/index.html

```diff
@@ -4,7 +4,7 @@
   <meta charset="UTF-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.5.0/css/all.css" integrity="sha384-B4dIYHKNBt8Bc12p+WXckhzcICo0wtJAoU8YZTY5qE0Id1GSseTk6S+L3BlXeVIU" crossorigin="anonymous">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.1/css/all.css" integrity="sha384-gfdkjb5BdAXd+lj+gudLWI+BXq4IuLW5IT+brZEZsLFm++aCMlF1V92rMkPaX4PP" crossorigin="anonymous">
```

Comment:
is this upgrade necessary?

Reply:
No, the upgrade has been reverse.

Reply:
When I remove it, the spinner fail to spin

Reply:
@sulenchy are you using the same spinner that's on develop? that should not be happening, make sure you're using the same spinner everyone else is using as well as the **same css to make it spin.**

Reply:
Yes I am using the same spinner